toolkit/geoview-compose/src/main/java/com/arcgismaps/toolkit/geoviewcompose/GeoViewScope.kt

```diff
@@ -134,6 +135,10 @@ public sealed class GeoViewScope protected constructor(private val geoView: GeoV
             allowCalloutRecomposition = true
             this.CalloutInternal(location, modifier, offset, rotateOffsetWithGeoView, colorScheme, shapes, content)
         }
+
+        SideEffect {
```

Comment:
Good catch, since `isCalloutBeingDisplayed` is defined outside the composition, it seems the right thing to do to update it in a SideEffect. I think we should add some comment though (also in the other function), how about?
```suggestion
        // The SideEffect is executed after every successful (re)composition. This means that it runs at the
        // end of the GeoView's content lambda from which this Callout function was called. Resetting at this point
        // allows us to run the callout code at subsequent recomposition but also to prevent multiple callouts from
        // being rendered within a single (re)composition pass.
        SideEffect {
```